articles/virtual-network/tutorial-create-route-table-portal.md

```diff
@@ -13,7 +13,7 @@ ms.custom: template-tutorial
 
 # Tutorial: Route network traffic with a route table using the Azure portal
 
-Azure routes traffic between all subnets within a virtual network, by default. You can create your own routes to override Azure's default routing. Custom routes are helpful when, for example, you want to route traffic between subnets through a network virtual appliance (NVA).
+Azure routes traffic between all subnets within a virtual network, by default. To customize your traffic routes, you shouldn't modify the default route but you should create your custom route, which overrides Azure's default routing. Custom routes are helpful when, for example, you want to route traffic between subnets through a network virtual appliance (NVA).
```

Comment:
```suggestion
By default, Azure routes traffic between all subnets within a virtual network. If you want to customize your traffic routes, create a custom route instead of modifying the default one. This custom route will override Azure's default routing. Use custom routes when you need to route traffic between subnets through a network virtual appliance (NVA), for example.
```